Makefile

```diff
@@ -551,16 +549,11 @@ test_e2e_xdc: bins
 cover_profile: bins
 	$Q mkdir -p $(BUILD)
 	$Q mkdir -p $(COVER_ROOT)
-	$Q echo "mode: atomic" > $(UNIT_COVER_FILE)
```

Comment:
is atomic default?  I think yes?
if not, I think it can be set via an arg for `go test`

Reply:
Yeah, i checked that the two generated cover files are the same, they are. So the `mode: atomic` is output by go test